From 78493e82d43e90430f2a61f6391db917c4420e74 Mon Sep 17 00:00:00 2001
From: Try Omarchy Release <actions@users.noreply.github.com>
Date: Mon, 7 Sep 2026 12:25:56 +0000
Subject: [PATCH] Refresh the guest package lock

enchant 2.8.19-1 -> 2.8.21-1
fcitx5 5.1.21-1 -> 5.1.22-1
fcitx5-qt 5.1.14-2 -> 5.1.15-1
hyprutils 0.14.1-1 -> 0.14.2-1
libheif 1.23.3-1 -> 1.23.4-1
libnautilus-extension 50.3-1 -> 50.3.1-1
linux 7.2.3.arch1-2 -> 7.2.3.arch1-3
nautilus 50.3-1 -> 50.3.1-1
pkgconf 3.0.6-1 -> 3.0.7-1
plocate 1.1.24-2 -> 1.1.25-1
---
 guest/packages.lock.json | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index 48156e9..e27b2e1 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -64,7 +64,7 @@
     "duktape": "2.7.0-7",
     "e2fsprogs": "1.47.4-1",
     "elfutils": "0.196-1",
-    "enchant": "2.8.19-1",
+    "enchant": "2.8.21-1",
     "exempi": "2.6.6-3",
     "exiv2": "0.28.9-1",
     "expat": "2.8.4-1",
@@ -72,9 +72,9 @@
     "fakeroot": "1:1.37.2-3",
     "fastfetch": "2.68.1-1",
     "fcft": "3.3.3-1",
-    "fcitx5": "5.1.21-1",
+    "fcitx5": "5.1.22-1",
     "fcitx5-gtk": "5.1.7-1",
-    "fcitx5-qt": "5.1.14-2",
+    "fcitx5-qt": "5.1.15-1",
     "fd": "10.5.0-3",
     "ffmpeg": "2:9.0.1-4",
     "fftw": "3.3.11-1",
@@ -168,7 +168,7 @@
     "hyprpicker": "0.4.7-4",
     "hyprsunset": "0.4.0-3",
     "hyprtoolkit": "0.5.4-5",
-    "hyprutils": "0.14.1-1",
+    "hyprutils": "0.14.2-1",
     "hyprwayland-scanner": "0.4.6-1",
     "hyprwire": "0.3.1-3",
     "iana-etc": "20260617-1",
@@ -269,7 +269,7 @@
     "libgsf": "1.14.58-1",
     "libgudev": "238-3",
     "libgxps": "0.3.2-5",
-    "libheif": "1.23.3-1",
+    "libheif": "1.23.4-1",
     "libhwasan": "16.2.1+r23+gd564253eb6c8-1",
     "libice": "1.1.2-1",
     "libidn2": "2.3.8-1",
@@ -298,7 +298,7 @@
     "libmodplug": "0.8.9.0-7",
     "libmpc": "1.4.1-1",
     "libmysofa": "1.3.5-1",
-    "libnautilus-extension": "50.3-1",
+    "libnautilus-extension": "50.3.1-1",
     "libndp": "1.9-1",
     "libnetfilter_conntrack": "1.1.1-1",
     "libnewt": "0.52.25-2",
@@ -418,7 +418,7 @@
     "libzip": "1.11.4-1",
     "licenses": "20240728-1",
     "lilv": "0.28.0-1",
-    "linux": "7.2.3.arch1-2",
+    "linux": "7.2.3.arch1-3",
     "linux-api-headers": "7.2-1",
     "llhttp": "9.3.1-1",
     "llvm-libs": "22.1.8-2",
@@ -450,7 +450,7 @@
     "mtdev": "1.1.7-1",
     "mujs": "1.3.9-1",
     "muparser": "2.3.5-2",
-    "nautilus": "50.3-1",
+    "nautilus": "50.3.1-1",
     "ncurses": "6.6-2",
     "neovim": "0.12.5-1",
     "nettle": "4.0-1",
@@ -504,8 +504,8 @@
     "pipewire-audio": "1:1.6.8-1",
     "pipewire-pulse": "1:1.6.8-1",
     "pixman": "0.46.4-1",
-    "pkgconf": "3.0.6-1",
-    "plocate": "1.1.24-2",
+    "pkgconf": "3.0.7-1",
+    "plocate": "1.1.25-1",
     "polkit": "127-3",
     "poppler": "26.08.0-1",
     "poppler-glib": "26.08.0-1",
-- 
2.55.0

